### ISSUE TITLE
time data does not match format '%Y-%m-%dT%H:%M:%S.%fZ'

### ISSUE THREAD

**Issue by benmazin** [rhydlewis/search-omnifocus#44]
Hi,

When attempting to search using .s or .sa does not find results, but shows a warning:

Error in workflow 'Search OmniFocus'
time data <timestamp> does not match format '%Y-%m-%dT%H:%M:%S.%fZ'

Using:
- Alfred 4.0.9 [1143]
- Search OmniFocus Alfred Workflow v2.1.0
- OmniFocus Pro - OmniFocus-3.6.x Tag: --tag OmniFocus/3.6/GM-v140.4

**Comment by rhydlewis** [comment]
I'm running the same version but can't reproduce this. However, I did spot another but similar  problem. I've also changed how the workflow tries to convert a date so that it doesn't fall over in the way you're seeing.

Have you migrated your database to the new format?

See [latest release](https://github.com/rhydlewis/search-omnifocus/releases/tag/v2.1.1) which should mean the workflow is useful again. Let me know how you get on.
